app/assets/javascripts/discourse/app/instance-initializers/auto-load-modules.js

```diff
@@ -18,24 +18,25 @@ export function autoLoadModules(container, registry) {
   });
 
   let context = {
-    siteSettings: container.lookup("service:site-settings"),
-    keyValueStore: container.lookup("service:key-value-store"),
-    capabilities: container.lookup("service:capabilities"),
-    currentUser: container.lookup("service:current-user"),
-    site: container.lookup("service:site"),
-    session: container.lookup("service:session"),
-    topicTrackingState: container.lookup("service:topic-tracking-state"),
+    siteSettings: owner.lookup("service:site-settings"),
+    keyValueStore: owner.lookup("service:key-value-store"),
+    capabilities: owner.lookup("service:capabilities"),
+    currentUser: owner.lookup("service:current-user"),
+    site: owner.lookup("service:site"),
+    session: owner.lookup("service:session"),
+    topicTrackingState: owner.lookup("service:topic-tracking-state"),
     registry,
   };
-  setOwner(context, container);
+  setOwner(context, owner);
 
   createHelperContext(context);
   registerHelpers(registry);
   registerRawHelpers(RawHandlebars, Handlebars);
 }
 
 export default {
-  name: "auto-load-modules",
   after: "inject-objects",
-  initialize: (container) => autoLoadModules(container, container.registry),
+  initialize: (owner) => {
+    autoLoadModules(owner, owner.__container__.registry);
```

Comment:
`owner.__container__.registry` is suspicious but it's again what the code already does now